release_notes_generator/utils/logging_config.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2024 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+This module contains a method to set up logging in the project.
+"""
+
+import logging
+import os
+import sys
+
+
+def setup_logging() -> None:
+    """
+    Set up the logging configuration in the project
+
+    @return: None
+    """
+    # Load logging configuration from the environment variables
+    is_verbose_logging: bool = os.getenv("INPUT_VERBOSE", "false").lower() == "true"
+    is_debug_mode = os.getenv("RUNNER_DEBUG", "0") == "1"
```

Comment:
For me verbose and debug are the same mode for logging. This is little confusing.

Reply:
- Our action(-s) provides the option to activate verbose logging. The user can decide whether to see the more detailed output in the step output.
- GitHub provides the feature to re-run the job with a debug flag. This is controlled by the RUNNER_DEBUG environment variable. Here, we are checking the value of this variable to automatically provide more detailed output.

So the user of our GH action will not be confused.